### ISSUE TITLE
Controls stay locked to side bar if using the AC button

### ISSUE THREAD

**Issue by xxredxpandaxx** [ChimeraOS/chimeraos#865]
I have ChineraOS 45 installed on my ROG Ally. If I bring up the right sidebar (the one with TDP control, notifications, etc.) and then close it with the Armory Crate button, my controls will still be locked to that menu. I have to close it by pressing B for proper functionality.

**Comment by pastaq** [comment]
Yeah, this is a bug in steam unfortunately so there isn't much we can do about it. Sometimes it works in the opposite way.
